tool_run_scripts/anvill.py

```diff
@@ -155,7 +155,6 @@ def make_tool_cmd(self):
             str(self.tmpout),
             "-stats_out",
             str(self.stats_file),
-            "-remove_next_pc_assignments",
```

Comment:
Not sure where this came from but the Anvill in `patchable-ir-main` doesn't recognise this.